### PR TITLE
[INTEL_MKL] Add `is_weight_const` Attr for QMatMulDequantize Op

### PR DESCRIPTION
**Pull request by retonym**
To add the missing `is_weight_const` attr in MklQuantizedMatMulAndDequantize registration.

### CODE REVIEW

File: tensorflow/core/common_runtime/mkl_layout_pass.cc [hunk 1]
```diff
@@ -2782,23 +2782,17 @@ void MklLayoutRewritePass::CopyAttrsQuantizedConv2D(const Node* orig_node,
...
-  DataType T1, T2, Toutput;
+  CopyAttrsAll(orig_node, nb, change_format);
+  
+  // Check and set filter attribute.
+  Node* filter_node = nullptr;
+  TF_CHECK_OK(orig_node->input_node(1, &filter_node));
+  nb->Attr("is_weight_const", filter_node->IsConstant());
 
-  // Get all attributes from old node.
+  // added "T" for facilitating MklToTf conversion.
```
Comment (penpornk) [inline tensorflow/core/common_runtime/mkl_layout_pass.cc:2792]: Nit: 
```suggestion
  // Add "T" for facilitating MklToTf conversion.
```

**Review by penpornk — CHANGES_REQUESTED** [review]
Sorry for the delay and thank you for the PR! I have one minor nit.

**Comment by retonym** [comment]
Thanks for the review.  Comments changed.

**Review by penpornk — APPROVED** [review]
Thank you!

**Review by gbaned — APPROVED** [review]

**Pull request merged**